backend/bitboxbase/rpcclient/config.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
```

Comment:
we are lucky this time since the diff is still easy, but in the future, please put renames and moves (files or code) in a separate commit, as it's much easier to review that it does not have any other changes (or if they do, what they are).